build.sbt

```diff
@@ -261,6 +266,22 @@ lazy val doc = project
     core
   )
 
+lazy val nodeFlywaySettings = {
+  lazy val DB_HOST = "localhost"
+  lazy val DB_NAME = "nodedb"
+  lazy val unittest = List(
+    Test / flywayUrl := s"jdbc:sqlite:${System.getenv("HOME")}/.bitcoin-s/unittest/nodedb.sqlite",
+    Test / flywayLocations := List("nodedb/migration"),
+    Test / flywayUser := "nodedb",
+    Test / flywayPassword := "",
+    flywayUrl := s"jdbc:sqlite:${System.getenv("HOME")}/.bitcoin-s/unittest/nodedb.sqlite",
+    flywayUser := "nodedb",
+    flywayPassword := ""
```

Comment:
Would it be an idea to do a env var here?

Reply:
We probably want to eventually, although with sqlite as the driver apparently it doesnt have users so it isn't really necessary now. I believe we could even remove the `nodedb` username setting.